File: week-07/day-1/Workshop 15.5.2023/seconds_in_ a_day.py
current_hours = 14;
current_minutes = 34;
current_seconds = 42;

# Write a program that prints the remaining seconds (as an integer) from a
# day if the current time is represented by the variables above

seconds = ((60-current_seconds) + ((60-current_minutes-1)*60) + (((24-current_hours-1)*60)*60))

print(seconds)

File: week-01/day-4/2023.04.06/src/SecondsInADay.java
public class SecondsInADay {
    public static void main(String[] args) {
        int currentHours = 14;
        int currentMinutes = 34;
        int currentSeconds = 42;

        // Write a program that prints the remaining seconds
        // (as an integer) from a
        // day if the current time is represented by
        // the variables above
        System.out.println(60-currentSeconds + ((60-1-currentMinutes)*60) + (24-1-currentHours)*60*60);

    }
}
